packages/calypso-products/src/main.ts

```diff
@@ -508,7 +508,7 @@ export function plansLink(
 }
 
 export function applyTestFiltersToPlansList(
-	planName: string,
+	planName: string | Plan,
```

Comment:
This took me a while to figure out, but because `planName` is passed to `getPlan()`, and the latter accepts a `Plan`, this does too. Otherwise the call to this function in `planSlugToPlanProduct()` might not have worked.